SPECS-EXTENDED/libguestfs/libguestfs.spec

```diff
@@ -775,8 +764,93 @@ fi
 mv README README.orig
 sed 's/@VERSION@/%{version}/g' < %{SOURCE4} > README
 
+# Re-enable upstream cache and local repos
+mkdir -pv /etc/yum.repos.d
+cp %{SOURCE9} /etc/yum.repos.d/allrepos.repo
+
+# Download appliance, since our chroot TDNF config does not have `keepcache=1`
```

Comment:
Double checking here:
Are we downloading stuff or using the local `rpm` cache to achieve this ?

Reply:
This uses the local RPM cache that is already mounted into the chroot, so we're not downloading anything from the network.